src/RGBEncoder.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * This file is a part of RGBEncoder library.
+ *
+ * Product page: https://amperka.ru/product/rgbencoder
```

Comment:
```suggestion
 * Product page: https://amperka.ru/product/troyka-rgb-encoder
```

Reply:
And, I suggest remove all links to the product page.

Reply:
Victor is unlikely to agree. This is a kind of ads.